packaging/helm/skipper-aws/templates/external-dns/aws-iam-role.yaml

```
@@ -0,0 +1,9 @@
+{{ if .Values.kube_aws_iam_controller }}
+apiVersion: zalando.org/v1
+kind: AWSIAMRole
+metadata:
+  name: external-dns-aws-iam-credentials
+  namespace: kube-system
```

This will allow users to install into another namespace if they choose.
```suggestion
  namespace: {{ .Release.Namespace }}
```